core/tasks/channels/cron.go

```diff
@@ -0,0 +1,94 @@
+package channels
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"github.com/nyaruka/gocommon/dates"
+	"github.com/nyaruka/mailroom/core/models"
+	"github.com/nyaruka/mailroom/core/msgio"
+	"github.com/nyaruka/mailroom/core/tasks"
+	"github.com/nyaruka/mailroom/runtime"
+)
+
+func init() {
+	tasks.RegisterCron("sync_old_seen_android_channels", &SyncOldSeenAndroidCron{})
+}
+
+type SyncOldSeenAndroidCron struct {
+	FCMClient msgio.FCMClient
+}
+
+func (s *SyncOldSeenAndroidCron) AllInstances() bool {
+	return true
+}
+
+func (s *SyncOldSeenAndroidCron) Next(last time.Time) time.Time {
+	return tasks.CronNext(last, time.Minute*5)
```

Comment:
RP has this at 10 minutes currently.. any reason to change?

Reply:
oops mistake I read that wrong